run_pacta_data_preparation.R

```diff
@@ -312,40 +312,68 @@ saveRDS(index_regions, index_regions_data_path)
 
 logger::log_info("Preparing financial data.")
 
+
 # read raw FactSet financial data, filter to unique rows, merge AR company_id,
 # merge PACTA sectors from AR data
 logger::log_info("Formatting and saving file: \"financial_data.rds\".")
+callr::r(
+  func = function(factset_financial_data_path, factset_issue_code_bridge, config) {
+    financial_data <-
+      pacta.data.preparation::prepare_financial_data(
+        data = readRDS(factset_financial_data_path),
+        issue_code_bridge = factset_issue_code_bridge
+      )
 
-financial_data <-
-  readRDS(factset_financial_data_path) %>%
-  pacta.data.preparation::prepare_financial_data(factset_issue_code_bridge)
+    saveRDS(
+      object = financial_data,
+      file = file.path(config[["data_prep_outputs_path"]], "financial_data.rds")
+    )
 
-saveRDS(
-  object = financial_data,
-  file = file.path(config[["data_prep_outputs_path"]], "financial_data.rds")
+    invisible()
```

Comment:
I believe `callr::r()` has to return something, but since we're using it for a side effect of saving the file it returns nothing/`NULL`, which I'd rather not automatically print to the console/logs, so I make it "invisible" meaning that it won't auto print the result of the function.

Reply:
Thanks :-)